GameServerLib/GameObjects/Buff.cs

```diff
@@ -123,6 +129,24 @@ public IStatsModifier GetStatsModifier()
             return BuffScript.StatsModifier;
         }
 
+        public void SetStatusEffect(StatusFlags flag, bool enabled)
+        {
+            // Loop over all possible status flags and assign them individually to the dictionary
+            foreach (StatusFlags enumFlag in Enum.GetValues(typeof(StatusFlags)))
```

Comment:
Enum.GetValues is a very slow function. Consider replacing it with something else.